### PR TITLE
Feature/post review api

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制]重要報導回顧-串接](https://app.asana.com/0/1203699264568808/1206721264374718/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/redux/src/actions/__test__/post-reviews.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import thunk from 'redux-thunk'
+
+import { mockPostReviewsData } from './mocks/post-reviews'
+
+const mockStore = configureStore([thunk])
+
+const apiOrigin = 'http://localhost:8080'
+
+describe('Test footprints action', () => {
+  describe(`Test action getUserFootprints`, () => {
```
Comment (Aylie-Chou) [inline packages/redux/src/actions/__test__/post-reviews.test.js:18]: user footprints -> post reviews

File: packages/redux/src/actions/__test__/post-reviews.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    const store = mockStore({
+      [fieldNames.origins]: {
+        api: apiOrigin,
+      },
+    })
+    afterEach(function() {
+      store.clearActions()
+      nock.cleanAll()
+    })
+    test('should dispatch an action created by `getUserFootprints` if fetching successed', () => {
```
Comment (Aylie-Chou) [inline packages/redux/src/actions/__test__/post-reviews.test.js:28]: same as [this comment](https://github.com/twreporter/twreporter-npm-packages/pull/533/files#r1591900800)

File: packages/redux/src/actions/__test__/post-reviews.test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          payload: {
+            statusCode: 200,
+            data: mockPostReviewsData,
+          },
+        }
+        expect(store.getActions()[0]).toEqual(requestExp)
+        expect(store.getActions()[1]).toEqual(successExp)
+      })
+    })
+    test('should dispatch an action created by `getUserFootprints` if fetching failed', () => {
```
Comment (Aylie-Chou) [inline packages/redux/src/actions/__test__/post-reviews.test.js:49]: same as [this comment](https://github.com/twreporter/twreporter-npm-packages/pull/533/files#r1591900800)

File: packages/redux/src/actions/post-reviews.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+ *  @typedef {Object} FailAction
+ *  @property {string} type - Action error type
+ *  @property {string} payload
+ *  @property {string} payload.statusCode - Response status code
+ *  @property {string} payload.message - Error message
+ *  @property {object} payload.error - Error object
+ */
+
+/**
+ * get user's donation history
```
Comment (Aylie-Chou) [inline packages/redux/src/actions/post-reviews.js:62]: could we also update the comment to post review version XD

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
mostly looking good~~
just some copy paste stuff to fix

File: packages/redux/src/actions/__test__/post-reviews.test.js (hunk 1 above)
Comment (liruchen32) [reply packages/redux/src/actions/__test__/post-reviews.test.js:18]: fixed

File: packages/redux/src/actions/__test__/post-reviews.test.js (hunk 2 above)
Comment (liruchen32) [reply packages/redux/src/actions/__test__/post-reviews.test.js:28]: fixed

File: packages/redux/src/actions/__test__/post-reviews.test.js (hunk 3 above)
Comment (liruchen32) [reply packages/redux/src/actions/__test__/post-reviews.test.js:49]: fixed

File: packages/redux/src/actions/post-reviews.js (hunk 4 above)
Comment (liruchen32) [reply packages/redux/src/actions/post-reviews.js:62]: fixed

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
漏網之魚！

File: packages/redux/src/actions/post-reviews.js (hunk 4 above)
Comment (liruchen32) [reply packages/redux/src/actions/post-reviews.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**